src/deluge/model/clip/audio_clip.cpp

```diff
@@ -253,7 +253,8 @@ Clip* AudioClip::cloneAsNewOverdub(ModelStackWithTimelineCounter* modelStackOldC
 	return newClip;
 }
 
-bool AudioClip::cloneOutput(ModelStackWithTimelineCounter* modelStack) {
+bool AudioClip::cloneOutput(ModelStackWithTimelineCounter* modelStack, OverDubType overdubNature) {
```

Comment:
Is it intentional you added overdubNature to all cloneOutput functions but don't supply it in the call or use it?

Reply:
No - remnant of an earlier approach that turned into a spaghetti mess